selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -380,46 +386,38 @@ void WifiManager::disconnect() {
   }
 }
 
-QDBusObjectPath WifiManager::pathFromSsid(const QString &ssid) {
-  QDBusObjectPath path;  // returns uninitialized path if network is not known
-  for (auto const& [conn_ssid, conn_path] : listConnections()) {
+QDBusObjectPath WifiManager::pathFromSsid(const QString &ssid, int &index) {
+//  QDBusObjectPath path;  // returns uninitialized path if network is not known
+  index = 0;
```

Comment:
what's the point of the index? why is it written even if the path isn't found?

Reply:
This was a hack to avoid iterating through `known_connections` twice in `forgetConnection`, and I didn't want to ignore using it in that one function for some unknown reason. I'll find a cleaner way to do this